tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -3027,6 +3044,47 @@ void handleQuarters(XPathParts parts, String value, List<String> examples) {
         examples.add(sdf.format(sample));
     }
 
+    /* Add relative date/time examples, choosing appropriate
+     * patterns as needed for relative dates vs relative times.
+     * Additionally, for relativeTimePattern items, ensure that
+     * numeric example corresponds to the count represented by the item.
+     */
+    private void handleRelative(
+            String xpath, XPathParts parts, String value, List<String> examples) {
+        String skeleton;
+        String type = parts.findAttributeValue("field", "type");
+        if (type.startsWith("hour")) {
+            skeleton = "Hm";
```

Comment:
We might want to make this "jm" or "Cm" to use hour cycle prefs based on locale. That can be done under [CLDR-18025](https://unicode-org.atlassian.net/browse/CLDR-18025) "Further work on examples for relative dates and times".